app/assets/javascripts/hyrax/save_work/validation_service.es6

```diff
@@ -0,0 +1,63 @@
+export default class ValidationService {
+  /**
+   * @param {Form} element
+   */
+  constructor(form, new_work) {
+    this.form = form
+    this.form.on('submit', (event) => { this.checkValidationService(event, new_work) })
+    this.setupAlert()
+    this.readOnlyAccession(new_work)
+  }
+
+  /**
+   * check the validation service before submit
+   *
+   */
+  checkValidationService(event, new_work) {
+      event.preventDefault()
+      this.closeAlert()
+      console.log('Checking validation service...');
+      var url = new_work ? '/validate_new' : '/validate_edit'
```

Comment:
Not sure if this code was pulled in externally, but common convention and some linting avoids `var` declarations.  Could use `const` or `let` if the value will change.